docs/docs/porting-part-of-a-html-site-to-gatsby.md

```diff
@@ -0,0 +1,569 @@
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>{children}</main>
+  </>
+)
+```
+
+The next step is to use that `<Layout>` component in the `index.js` page file. Gatsby itself provides a number of core building blocks, `<Link>` is one of them. `<Link>` is imported at the top of the file to replace the `<a>` tags. The Link component takes a `to` prop as opposed to the `href` of an anchor tag. Using the `<Link>` component enables Gatsby to make optimizations like prefetching page content just before a user clicks a link.
+
+```jsx:title=/gatsby-site-section/src/pages/index.js
+import React from "react"
+import Layout from "../components/Layout"
+import { Link } from "gatsby"
+
+export default () => (
+  <Layout>
+    <h2>These are our staff:</h2>
+    <ul>
+      <li>
+        <Link to="/ella-arborist">Ella (Arborist)</Link>
+      </li>
+      <li>
+        <Link to="/sam-surgeon">Sam (Tree Surgeon)</Link>
+      </li>
+      <li>
+        <Link to="/marin-leafer">Marin (Leafer)</Link>
+      </li>
+    </ul>
+  </Layout>
+)
+```
+
+### Porting other section pages
+
+Now it's time for the work to really pay off! Ella's page is a matter of using your `<Layout>` component again and copying in the main content. Don't forget to change `class` to `className`! A `staffName` prop can be passed to `<Layout>` this time to change the dynamic page title, passing props is similar to an attribute on a HTML element:
+
+```jsx:title=/gatsby-site-section/src/pages/ella-arborist.js
+import React from "react"
+import Layout from "../components/Layout"
+import { Link } from "gatsby"
+
+export default () => (
+  {/* highlight-start */}
+  <Layout staffName="Ella">
+  {/* highlight-end */}
+    <h2>Ella is an excellent Arborist. We guarantee it.</h2>
+    <div className="bio-card">
+      <img
+        alt="Comically crude stick person sketch"
+        src="https://gatsby-html-partial-assets.s3.eu-west-2.amazonaws.com/person.png"
+      />
+      <p>Ella</p>
+    </div>
+  </Layout>
+)
+```
+
+The other 2 pages for Marin and Sam can now be made with a similar structure. Maybe you are even thinking about another component for the Bio Card!
+
+## Build
+
+With your new Gatsby application complete, it's time to integrate it into your existing HTML website. If you were to build the Gatsby application into static files now and upload them in the place of the existing HTML files, the paths of the links made by Gatsby would not be correct. There are a couple of Gatsby configuration options to fix this.
+
+### Path Prefix
+
+The `pathPrefix` option in `/gatsby-config.js` tells Gatsby the path at which the build output will be served from. Links to other pages and to the generated JavaScript files that provide [progressive enhancements](/docs/glossary/#progressive-enhancement) for the user will be prefixed with the `pathPrefix` value. This is the config addition for the example Gatsby application, served at the `/who` path:
```

Comment:
What do you mean by "generated JavaScript files" here? Isn't that covered by pages? "other pages" could also refer to the pages that haven't been touched by Gatsby, and if they were prefixed with `/who`, might not be correct.

```suggestion
The `pathPrefix` option in `/gatsby-config.js` tells Gatsby the path at which the build output will be served from. For example, links to other pages within your Gatsby site will be prefixed with the `pathPrefix` value. Here is some config code to tell Gatsby it should only handle and care about the `/who` path for now:
```

Reply:
I was trying to acknowledge the many new JavaScript files that we are asking them to host. My thinking is that the audience here will be used to just having a collection of known HTML and CSS files to host. Thinking now that it should be addressed directly with a section if at all though. If assetPrefix is not part of the example I think it's less important to address.

Will pick this up in the discussion below where you brought up assetPrefix.